yarn.lock

```diff
@@ -2975,7 +2975,14 @@ glob@~7.0.6:
     once "^1.3.0"
     path-is-absolute "^1.0.0"
 
-global@4.3.2, global@^4.3.0, global@^4.3.1, global@^4.3.2, global@~4.3.0:
+global@4.3.0:
```

Comment:
@mbertrand am I right in thinking that? I think on a recent PR you said that the file had to be hand-edited for some reason but I don't rememeber the specifics

Reply:
Right, I manually edited `yarn.lock` because videojs was being included twice with different versions after adding the youtube plugin, which caused problems.  The changes here didn't affect that part, but AFAIK they're not necessary to fix the layout issue.